src/majordomo/include/majordomo/RestBackend.hpp

```diff
@@ -742,10 +742,10 @@ struct RestBackend<Mode, VirtualFS, Roles...>::RestWorker {
     bool respondWithLongPoll(const httplib::Request &request, httplib::Response &response, const std::string_view &_service) {
         // TODO: After the URIs are formalized, rethink service and topic
         auto                     split = std::ranges::find(_service, '/');
-        std::string_view         service(_service.begin(), split);
-        std::string_view         topic(split, _service.end());
+        std::string              service(_service.begin(), split);
+        std::string              topic = "/"s + service + std::string(split, _service.end());
```

Comment:
```suggestion
        std::string service = _service;
        std::string     topic = "/"s + _service + "?" + request.paramsString;
```
This is what I would expect here.

Reply:
But this might need other changes since we do not want to include the long polling params into the topic... so I'd just merge this as is to keep giulio working on the UI part and wait for this to be resolved separately. Just to keep in mind that/how the mechanics might change and that we cannot yet rely on query parameters working over the rest backend.

Reply:
It is still not quite clear to me what the roles of all the various parts of an uri are, so i tried to change this code as little as possible, but to me your suggested change looks like it makes sense. Maybe @ivan-cukic can chime in?